src/transactions/TransactionFrame.cpp

```diff
@@ -1919,9 +1919,6 @@ TransactionFrame::parallelApply(
             ledgerInfo.getLedgerVersion() >=
             config.LEDGER_PROTOCOL_MIN_VERSION_INTERNAL_ERROR_REPORT;
 
-        auto& opTimer =
```

Comment:
Was this removal intentional?

Reply:
Ya, it's an unused variable. Is it intended to call Timescope()?